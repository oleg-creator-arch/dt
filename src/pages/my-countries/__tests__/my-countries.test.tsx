import { renderComponentWithStore } from '@/shared/lib/test-helpers';
import MyCountries from '..';

it('renders correctly home', () => {
  const { container } = renderComponentWithStore({
    Component: MyCountries,
    preloadedState: {
      infoCountries: {
        listCountries: [],
        fullInfoCountry: [],
        isFullInfoCountry: false,
        isLoading: false,
        infoErrorResponse: '',
      },
      infoUser: {
        userName: '',
        listFavoriteCountries: [],
        isAuthentication: false,
        infoModal: { isActiveModal: false, formModal: 'login' },
      },
    },
  });
  expect(container).toMatchSnapshot();
});
