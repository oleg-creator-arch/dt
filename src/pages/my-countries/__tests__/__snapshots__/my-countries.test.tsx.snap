// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly home 1`] = `
<div>
  <div
    class="favorite-container"
  >
    <div
      class="empty-favorite-container"
    >
      <div
        class="info-about-empty-favorite"
      >
        <div>
          The list of favorite countries is empty
        </div>
        <div
          class="choose-country"
        >
          Choose a country
        </div>
      </div>
    </div>
  </div>
  <div
    id="portal"
  />
</div>
`;
